Apps/CocoaPods-FCM/src/View/CustomAttributeView.swift

```diff
@@ -36,13 +36,19 @@ struct CustomAttributeView: View {
                 }.padding([.vertical], 40)
 
                 ColorButton("Send \(attributeTypeName) attributes") {
-                    showConfirmationAlert = true
+                    var alertMessage = "\(attributeTypeName.capitalized) attribute sent successfully!"
+
+                    if name.isEmpty || value.isEmpty {
+                        alertMessage += "\n\n Note: Empty attribute name or value might result in expected behavior with the SDK."
```

Comment:
Do you mean "unexpected" behavior with the SDK ?

Reply:
Yup! Thanks for catching that. 

```suggestion
                        alertMessage += "\n\n Note: Empty attribute name or value might result in unexpected behavior with the SDK."
```